app/src/main/java/com/connorlay/knitgrid/adapters/KnitGridAdapter.java

```diff
@@ -0,0 +1,90 @@
+package com.connorlay.knitgrid.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.connorlay.knitgrid.R;
+import com.connorlay.knitgrid.models.Stitch;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+/**
+ * Created by Greg on 3/17/2016.
+ */
+public class KnitGridAdapter extends RecyclerView.Adapter<KnitGridAdapter.StitchRecycleViewHolder> {
```

Comment:
This class name is kinda ambiguous... what about something like `PatternLegendRecyclerViewAdapter`?